src/elements/content-sidebar/Sidebar.js

```diff
@@ -111,8 +112,18 @@ class Sidebar extends React.Component<Props, State> {
             this.setForcedByLocation();
             this.setState({ isDirty: true });
         }
+
+        // Reset the current version id if the wrapping versions route is no longer active
+        if (onVersionChange && this.getVersionsMatchPath(prevLocation) && !this.getVersionsMatchPath(location)) {
```

Comment:
ContentSidebar already does a lot. I'm wondering if this would be better suited to live in `SidebarPanels`, since that's where we manage all of this already.